apps/frontend/src/components/Stopwatch.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import { ArrowPathIcon, ClockIcon } from "@heroicons/react/24/outline";
+import { formatTime as formatStopwatchTime } from "@/utils/format/timeFormat";
+
+type StopwatchProps = {
+};
+
+const Stopwatch: React.FC<StopwatchProps> = () => {
+  const [showStopwatch, setStopwatch] = React.useState(false);
+  const [time, setTime] = React.useState(0);
```

Comment:
We could improve this timer, by making it persistent, so that user can come back and continue with the timing?

Reply:
Sure I created #52, will improve it another time